localization/english/CWP/CWP_resolutions_l_english.yml

```diff
@@ -26,8 +26,20 @@
   concept_un_headquarters_resolution: "UN Headquarters Resolution"
   concept_decolonization_resolution: "Convention on Decolonization"
   concept_apartheid_resolution: "Convention on Apartheid"
-  concept_environmental_modification_resolution: "ENMOD"
+  concept_environmental_modification_resolution: "ENMOD""
   concept_sustainable_development_resolution: "Sustainable Development"
   concept_unaec_resolution: "UNAEC"
   concept_unga_partition_resolution: "UNGA Partition Resolution"
   concept_resolution_on_korea: "Resolution on Korea"
+  
+  concept_unclos2_desc: "In 1960, the United Nations held the second Conference on the Law of the Sea ('UNCLOS II'); however, the six-week Geneva conference did not result in any new agreements. Generally speaking, developing nations and third world countries participated only as clients, allies, or dependents of the United States or the Soviet Union, with no significant voice of their own."
+  concept_unclos3_desc: "The issue of varying claims of territorial waters was raised in the UN in 1967 by Arvid Pardo of Malta, and in 1973 the Third United Nations Conference on the Law of the Sea convened in New York. In an attempt to reduce the possibility of groups of nation-states dominating the negotiations, the conference used a consensus process rather than majority vote. With more than 160 nations participating, the conference lasted until 1982. The resulting convention came into force on 16 November 1994, one year after the 60th state, Guyana, ratified the treaty.\nThe convention introduced a number of provisions. The most significant issues covered were setting limits, navigation, archipelagic status and transit regimes, exclusive economic zones (EEZs), continental shelf jurisdiction, deep seabed mining, the exploitation regime, protection of the marine environment, scientific research, and settlement of disputes."
+  concept_anti_corruption_resolution_desc: "The United Nations Convention Against Corruption (UNCAC) is the only legally binding international anti-corruption multilateral treaty. Negotiated by member states of the United Nations (UN) it has been adopted by the UN General Assembly in October 2003 and entered into force in December 2005. The treaty recognises the importance of both preventive and punitive measures, addresses the cross-border nature of corruption with provisions on international cooperation and on the return of the proceeds of corruption. The UN Office on Drugs and Crime (UNODC) in Vienna serves as Secretariat for the UNCAC. UNCAC's goal is to reduce various types of corruption that can occur across country borders, such as trading in influence and abuse of power, as well as corruption in the private sector, such as embezzlement and money laundering. Another goal of the UNCAC is to strengthen international law enforcement and judicial cooperation between countries by providing effective legal mechanisms for international asset recovery. "
```

Comment:
At this line is where the yaml indentation issue begins. eg:

```yaml
l_english:
 key: "content"
 key: "content"
```

Reply:
Cleaned it up